src/flows/braze/ensure_pocket_users_have_email_alias_flow.py

```diff
@@ -0,0 +1,149 @@
+from prefect import Flow, context, task, Parameter
+import datetime
+import re
+from typing import Any, Dict, List, Tuple
+from dataclasses import dataclass
+
+from api_clients.pocket_snowflake_query import OutputType, PocketSnowflakeQuery
+from common_tasks.braze import mask_email_domain_outside_production
+from common_tasks.mapping import split_in_chunks
+from utils.flow import get_s3_result, get_flow_name
+from api_clients.braze import models
+from api_clients.braze.client import (
+    BrazeClient,
+    IDENTIFY_USER_ALIAS_LIMIT
+)
+from api_clients.braze.pocket_config import EMAIL_ALIAS_LABEL
+
+from utils.iteration import chunks
+
+FLOW_NAME = get_flow_name(__file__)
+
+EXTRACT_QUERY = """
+SELECT
+    braze_id,
+    external_id,
+    email
+FROM "{table_name}"
+WHERE external_id is not null 
+    and email is not null
+    and user_aliases is null
```

Comment:
This flow seems good to me! I do think before running, we want to make sure that Census actually did set an alias for the profiles that didn't have duplicates in the first place (and weren't part of the EN PH migration).

Otherwise, though those profiles may not currently have duplicates, they might get them in the future (bringing us back to our current predicament).

Reply:
Actually, I think we can verify this by running a query over the data you exported:
```
SELECT
    count(braze_id),
    count(user_aliases),
    count(email),
    count(external_id)
FROM DEVELOPMENT.daniel.BRAZE_USERS_11_01_2022
;
```

This shows that there are ~42M profiles in Braze that have aliases, which is _definitely_ more than the ~4-5M we would have migrated for EN Pocket Hits! So Census must have correctly created aliases for all of the profiles that don't have duplicate, alias only records.